resize.go

```diff
@@ -180,7 +182,18 @@ func transformImage(image *C.VipsImage, o Options, shrink int, residual float64)
 	}
 
 	if o.Force || residual != 0 {
-		image, err = vipsAffine(image, residualx, residualy, o.Interpolator)
+		xshrink := 1.0 / residualx
```

Comment:
@oertekin 
Please update to Go naming convention as per: https://golang.org/doc/effective_go.html